gui/functionality/idact_notebook.py

```diff
@@ -114,6 +114,8 @@ def deploy_notebook(self):
                 notebook.open_in_browser()
                 sleep_until_allocation_ends(nodes=nodes)
         except:  # noqa, pylint: disable=broad-except
+            self.window = ErrorApp()
+            self.window.show()
             if log is not None:
```

Comment:
You can delete this if about log and the log itself. It is defined as none and never changed. And now we have something to put in the body of except so it isn't needed.

Reply:
Ok, I deleted it.